articles/search/search-security-overview.md

```diff
@@ -166,16 +166,18 @@ Currently, the only external resource that a search service writes customer data
 
 ### Exceptions to data residency commitments
 
-Although customer data isn't stored outside of your region, object names (considered as customer data), will appear in the telemetry logs used by Microsoft Support to troubleshoot your service issues. Telemetry logs include names of indexes, indexers, data sources, skillsets, containers, and key vault store.
+Non-personal data received from an organization, or generated from the use of the product, such as component names, will be stored and processed outside your selected region or location. This data will appear in the telemetry logs used by Microsoft to provide support for the service. Component names include names of indexes, indexers, data sources, skillsets, containers, and key vault store.
```

Comment:
I deleted my previous comment so that it isn't confused with this one. For line 169, I don't know what "non-personal data received from an organization" means. Not sure what's meant by "non-personal data", or who is doing the "receiving" (guessing it's MSFT), or who the "organization" is, although I think you mean "customer".

If you say "Non-personal data... will be stored and processed outside of your selected region or location", you definitely need to say what you mean by non-personal data because practically speaking, most data is non-personal data unless your search index has data about people.  It doesn't really seem like "an exception" anymore.

Finally, the scope of the exception is no longer clear.  I thought this was about object names showing up in telemetry logs, but with a more generalized phrasing, and I'm wondering what else besides object names are stored and processed outside of the customer's region?

Reply:
To address both, comments, maybe it would make sense to move the "Important" recommendation up to simplify the message and clarify that object names should not contain personal data in one place.  This is what I'd propose...

_Object names will be stored and processed outside of your selected region or location. Customers shouldn't place any personal data in these fields or create applications designed to store personal data in these fields. This data will appear in the telemetry logs used by Microsoft to provide support for the service. Object names include names of indexes, indexers, data sources, skillsets, containers, and key vault store._

Then we can remove the Important section.

Reply:
I like it!  Makes perfect sense. I agree you don't need the Important block anymore.

Reply:
@HeidiSteen, besides of the definitions, our main focus is that customers understand that only object names will be stored and processed outside their location, so they should avoid placing sensitive information.  Thank you @mrcarter8, I agree with this simpler version too.